core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -521,11 +521,11 @@ func (c *coordinator) filterUnfulfilledCallbacks(
 		// Check if there is room left in the batch. If there is no room left, the coordinator
 		// will keep iterating, until it either finds a callback in a subsequent output height that
 		// can fit into the current batch or reaches the end of the sorted callbacks slice.
-		if c.coordinatorConfig.BatchGasLimit-currentBatchGasLimit < (r.Callback.GasAllowance.Int64() + c.coordinatorConfig.CallbackOverhead) {
+		if c.coordinatorConfig.BatchGasLimit-currentBatchGasLimit < (int64(r.GasAllowance) + c.coordinatorConfig.CallbackOverhead) {
```

Comment:
nit: if this is now an int64 does it need type casting?

Reply:
It seems like it's a `uint32` now. https://github.com/smartcontractkit/chainlink/pull/8049/files#diff-e3f18c3afd30cd2ce069d58e79d0a04f5cff87802e746b179c3437dd7bcb6f5eR1670

Reply:
yeah it's a `uint32` in the `requestRandomnessFulfillment` [here](https://github.com/smartcontractkit/threshold-vrf/blob/master/contracts/VRFCoordinator.sol#L63). Seems like gasAllowance can be `uint32` everywhere.